internal/proto/controller/api/services/v1/account_service.proto

```diff
@@ -13,68 +13,136 @@ import "protoc-gen-openapiv2/options/annotations.proto";
 
 option go_package = "github.com/hashicorp/boundary/internal/gen/controller/api/services;services";
 option (custom_options.v1.domain) = "auth";
+// Global definitions for our API, have to go in one of the protobuf files.
+// Putting it in the file that's alphabetically sorted first means it maintains
+// the alphabetic ordering of services.
+option (grpc.gateway.protoc_gen_openapiv2.options.openapiv2_swagger) = {
+  info: {
+    version: "placeholder-version" // Replaced at generation time by version/VERSION contents
+    title: "Boundary controller HTTP API"
+    description:
+      "Welcome to the Boundary controller HTTP API documentation. "
+      "This page contains the complete specification for all the API endpoints exposed by the Boundary controller.\n\n"
+      "To get started, look for the service you want to interact with in the sidebar to the left. "
+      "Each resource in Boundary, such as accounts and credential stores, have their own service. "
+      "Each service contains all the API endpoints supported for the resource."
+    contact: {
+      name: "HashiCorp Boundary"
+      url: "https://www.boundaryproject.io/"
+    };
+    license: {
+      name: "Business Source License 1.1"
+      url: "https://github.com/hashicorp/boundary/blob/main/LICENSE"
+    };
+  }
+  host: "your-boundary-hostname.com"
```

Comment:
We could have something like `localhost:9200` here as well, not sure what makes this the most obvious that it depends on their Boundary deployment?